.pre-commit-config.yaml

```diff
@@ -5,11 +5,19 @@ repos:
   rev: v1.9.0
   hooks:
   - id: reorder-python-imports
+- repo: https://github.com/asottile/pyupgrade
+  rev: v2.7.2
+  hooks:
+  - id: pyupgrade
+    args: [--py38-plus]
```

Comment:
Shouldn't that be `--py36-plus`?

I couldn't find authoritative list of what pyupgrade changes for each Python version, but I tried making sense out of their source code and seems they only consider two things for Python above 3.6:

* `from __future__ import generator_stop`
* `functools.lru_cache` decorator doesn't need to be called

We don't use either in Airgun, so both flags should produce same results for us right now. But I'm little worried this might result in tool changing our syntax in backward-incompatible way sometime in the future, when they decide more changes are OK in 3.8.

Reply:
If we're having to stick to 36 because of the IQE integration, then yes.

Reply:
Dialed this back to 36plus.